records_mover/db/vertica/unloader.py

```diff
@@ -47,7 +47,7 @@ def unload(self,
                table: str,
                unload_plan: RecordsUnloadPlan,
                directory: RecordsDirectory) -> Optional[int]:
-        if not self.s3_available():
+        if not self.s3_export_available():
```

Comment:
I'm separating out the concepts of "does the database have the right add-ons installed to export to S3" and "do we have a temporary S3 bucket available".

Even if we don't have a *temporary* S3 bucket available, we may be able to export straight to the destination S3 bucket!